internal/captain/command.go

```diff
@@ -880,3 +918,30 @@ func childCommands(cmd *Command) string {
 
 	return fmt.Sprintf("\n\nAvailable Commands:\n%s", table.Render())
 }
+
+func (c *Command) LogArgs() {
+	children, err := c.FindChildren(os.Args[1:])
```

Comment:
I'm pretty sure `FindChild()` already achieves the desired effect you're looking for. And it passes through cobra, guaranteeing that we get the right command. From there I think you could use `command.cobra.CommandPath()`, which should give you the full path.